lib/request.js

```diff
@@ -84,7 +84,7 @@ module.exports = {
   get href() {
     // support: `GET http://example.com/foo`
     if (/^https?:\/\//i.test(this.originalUrl)) return this.originalUrl;
-    return this.origin + this.originalUrl;
+    return `${this.origin}${this.originalUrl}`;
```

Comment:
I much prefer the original. Formatting strings are meant for when you have a variable as part of a larger surrounding string, not for concatenation.